subsys/bluetooth/host/br.c

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright 2024 NXP
```

Comment:
Could you follow the same format as for other copyright lines, i.e. "Copyright (c) <year> <company>? (you have the "(c)" missing right now.

Reply:
NXP usually don't add that

Reply:
I will remove this line.

Reply:
@Thalley is correct. Our NXP legal guidance is how @lylezhu2012 is doing it. Don't ask me why.. we just do what our legal says we need to do ;-)